test/integration/evaluate.test.js

```diff
@@ -42,3 +45,70 @@ describe('evaluate', () => {
     })
   );
 });
+
+describe('evaluateAsync', () => {
+  if (browserName === 'phantomjs') {
+    xit("skipping tests because browser phantomjs doesn't support alerts");
+    return;
+  }
+
+  before(browser.beforeHook());
+
+  before(() => browser.loadPage('/'));
+
+  it(
+    'runs JavaScript passed as a Function',
+    co.wrap(function*() {
+      assert.equal(
+        6,
+        // eslint-disable-next-line prefer-arrow-callback
```

Comment:
if we're skipping phantom anyway, we can just go ahead and use arrow functions for all these